packages/ui/src/working-groups/constants.ts

```diff
@@ -0,0 +1,14 @@
+export enum OpeningStatuses {
+  UPCOMING = 'OpeningStatusUpcoming',
+  OPEN = 'OpeningStatusOpen',
+  FILLED = 'OpeningStatusFilled',
+  CANCELLED = 'OpeningStatusCancelled',
+}
```

Comment:
I was thinking about this one. I rather not use enums (minor issues in the runtime). But here I think that this actually a really nice use case for string enum.